clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/CredentialsController.groovy

```diff
@@ -59,6 +59,11 @@ class CredentialsController {
     return accountDetail
   }
 
+  @RequestMapping(value = "/disabled", method = RequestMethod.GET)
```

Comment:
Not a huge fan of a `/disabled` endpoint.

It should be possible to iteratively make callers of `/credentials` aware of the `enabled` attribute?

Reply:
Sure. I just figured this would be less work. But you mean to always include all accounts? Or to make a query param or something at the `/credentials` endpoint?

Reply:
I've removed it